### PR TITLE
Improve logging again

### PR DESCRIPTION
**Pull request by insacuri**
**What is the context of this PR?**

Changed a number of "string".format() to f"string" to make it read easier and shorten the length of log lines.
Added additional logging to aid in debugging

**How to review**

Any notes for the reviewer  (include screenshots if appropriate).

### CODE REVIEW

File: tests/app/test_app.py [hunk 1]
```diff
@@ -163,7 +163,7 @@ def test_missing_thread_id_does_not_cause_exception(self):
             self.client.post(url, data=json.dumps(test_message), headers=self.headers)
             self.assertTrue(True)  # i.e no exception
         except Exception as e:
-            self.fail("post raised unexpected exception: {0}".format(e))
+            self.fail(f"post raised unexpected exception: {e}")
```
Comment (JamesGardiner) [inline tests/app/test_app.py]: Could you log the exception using logger.exception, then just log the message text in self.fail using e.message?

**Comment by insacuri** [comment]
@AndrewTorrance at the moment when you get the message list, you get a log line with the JWT encrypt, and then nothing until you get the 200 back.  Debugging this is impossible because it's like a ghost town in places

File: tests/app/test_app.py (hunk 1 above)
Comment (JamesGardiner) [reply tests/app/test_app.py]: @AndrewTorrance I don't think so, as the test is for not raising a generic exception.

Comment (JamesGardiner) [reply tests/app/test_app.py]: Approving this based on the above test being refactored via a github issue.

**Pull request merged**